subsys/bluetooth/controller/ll_sw/ull_master.c

```diff
@@ -160,7 +160,7 @@ u8_t ll_create_connection(u16_t scan_interval, u16_t scan_window,
 
 	conn_lll->data_chan_count =
 		ull_conn_chan_map_cpy(conn_lll->data_chan_map);
-	util_rand(&hop, sizeof(u8_t));
+	lll_trng_get(&hop, sizeof(u8_t));
```

Comment:
This most likely does not need true randomness, but creating a connection is probably rare enough to not matter that much.

Reply:
Probably not. there might be other instances too.
I would suggest someone go through and change those that can be changed, but not as part of this PR :)